config/webpack.prod.config.js

```diff
@@ -61,13 +68,25 @@ module.exports = Merge.smart(commonConfig, {
         }),
       },
       {
+        // Font-Awesome and OpenSans is already loaded and available in Studio
         test: /\.(woff2?|ttf|svg|eot)(\?v=\d+\.\d+\.\d+)?$/,
-        loader: 'file-loader',
+        loader: 'null-loader',
       },
     ],
   },
   plugins: [
-    new ExtractTextPlugin('studio-frontend.min.css'),
+    new webpack.optimize.CommonsChunkPlugin({
+      name: 'vendor',
+      minChunks: module => module.context && module.context.includes('node_modules'),
+    }),
+    new webpack.optimize.CommonsChunkPlugin({
+      name: 'manifest',
+      minChunks: Infinity,
+    }),
```

Comment:
This webpack incantation is explained here: https://webpack.js.org/plugins/commons-chunk-plugin/#combining-implicit-common-vendor-chunks-and-manifest-file